lib/src/page_list_viewport_gestures.dart

```diff
@@ -454,57 +670,85 @@ class DeprecatedPanAndScaleVelocityTracker {
       : null;
 }
 
+double _repeatedSwipeVelocityIncreaseScalar(int numberOfRepeatedAcceleratedSwipes) {
+  // The function takes the number of the repeated swipes already considered in the repeated swipe acceleration sequence
+  // and as returns the launch velocity scalar for the momentum simulation.
+  // The model for acceleration due to repeated input assumes this
+  // model: startValue+\frac{endValue-startValue}{1+e^{-k(x-transitionValue)}}
+  const double transitionValue = 9; // where the function takes it's intermediate value
+  const double k = 0.5; // how quickly the shift happens (smaller is slower)
+  const double startValue = 1;
+  const double endValue = 14;
+  return startValue + (endValue - startValue) / (1 + exp(-k * (numberOfRepeatedAcceleratedSwipes - transitionValue)));
+}
+
+double _repeatedSwipeDragIncreaseScalar(int numberOfRepeatedAcceleratedSwipes) {
+  // The function takes the number of the repeated swipes already considered in the repeated swipe acceleration sequence
+  // and as returns the drag increase scalar for the momentum simulation.
+  // model: startValue+\frac{endValue-startValue}{1+e^{-k(x-transitionValue)}}
+  const double transitionValue = 5; // where the function takes its intermediate value
+  const double k = 0.8; // how quickly the shift happens (smaller is slower)
+  const double startValue = 1;
+  const double endValue = 0.5;
+  return startValue + (endValue - startValue) / (1 + exp(-k * (numberOfRepeatedAcceleratedSwipes - transitionValue)));
+}
+
 class PanningFrictionSimulation implements PanningSimulation {
   // Dampening factors applied to each component of a [FrictionSimulation].
   // Larger values result in the [FrictionSimulation] to accelerate faster and approach
   // zero slower, giving the impression of the simulation being "more slippery".
   // It was found through testing that other scroll systems seem to be use different dampening
   // factors for the vertical and horizontal components.
-  static const kVerticalDrag = 0.095;
-  static const kHorizontalDrag = 0.0425;
+  static const horizontalDragCoefficient = 250.0;
+  static const verticalDragCoefficient = 300.0;
+  static const staticFrictionCoefficient = 20.0;
+  // Mass is a redundant parameter, the ratio of m/c, mass to drag is important! Don't change:
```

Comment:
First, the comment seems to be contradictory. It says "Mass is a redundant parameter" and yet it also says "mass to drag is important!". Which is it?

Second, a comment should never say "Don't change". Such a statement is counterproductive. Instead, you should provide sufficient explanation, contextualizing the property, such that a developer knows when he should change it, and how to validate the result of changing it.